package.json

```diff
@@ -30,6 +30,7 @@
     "@remix-run/serve": "^2.7.1",
     "@shopify/app": "^3.57.1",
     "@shopify/cli": "^3.57.1",
+    "@shopify/app-bridge-react": "^4.1.2",
```

Comment:
Should we also drop the app bridge types package?

Reply:
Good catch, the types are a dep in ABR4 so they can be removed from the package here